static/app/views/replays/detail/network/index.tsx

```diff
@@ -325,14 +328,21 @@ const NetworkFilters = styled('div')`
   }
 `;
 
-const Item = styled('div')<{center?: boolean; color?: ColorOrAlias; numeric?: boolean}>`
+const Item = styled('div')<{
+  center?: boolean;
+  color?: ColorOrAlias;
+  statusError?: boolean;
+  numeric?: boolean;
+  statusCode?: boolean;
```

Comment:
This reads like a value, it could be prefixed with `is` so it's clearer that it's a bool

```suggestion
  isStatusCode?: boolean;
```